tests/opwen_email_server/integration/test_celery.py

```diff
@@ -38,3 +40,13 @@ def test_send_message(self):
                 routing_key=self.routing_key,
                 declare=[self.queue],
             )
+
+    def test_celery_task_routes(self):
```

Comment:
Can this test be named more meaningfully to reflect its intent? This will make it easier to figure out what may be wrong when the test breaks in the future and people may not have looked at it in a while. E.g. `test_all_celery_tasks_are_routed` or something like that.